WeatherApp/WeatherApp/CityScreen/CityScreenViewModel.swift

```diff
@@ -6,22 +6,26 @@ class CityScreenViewModel: ObservableObject {
     private let weatherService: WeatherServiceProtocol
 
     @Published var city: String
-    @Published var weather: WeatherModel?
+    @Published var weather: CurrentWeatherResponse?
```

Comment:
private(set) both `@Published `properties

Make sure to check other view models as well and fix acorrdingly